questions/testing-questions.md

```diff
@@ -4,3 +4,4 @@
 * What tools would you use to test your code's functionality?
 * What is the difference between a unit test and a functional/integration test?
 * What is the purpose of a code style linting tool?
+* How do you ensure your code is fit for purpose?
```

Comment:
I had to look this phrase up because it's uncommon to my American ear. 

[Is "fit for purpose" fit for purpose?](http://www.macmillandictionaryblog.com/fit-for-purpose)

I like this idea of this question, I'm not sure that one brief sentence is enough of a setup to get the open-ended response you're looking for.

Reply:
Ah good point, I see what you mean, I'll have a think today about a more international version 👍

Reply:
Hey @roblarsen sorry for late response, I've had another crack at that question,  I ended up going with the words that were in that useful link you sent, hopefully that should make sense whatever side of the pond you are on 🇬🇧🇺🇸.

Let me know if you think it's of any use, as it does still have some ambiguity (I'm equally happy to close it down and get it out of the queue)